### PR TITLE
Item description changes

### PR DESCRIPTION
**Pull request by flipog**
Humor is hard and I'm well aware that what is funny to me may not be as funny for other people. With this is mind, what I tried to do was replicate some of the humor I see around me... Maybe I failed hard, but there it is ;)

### CODE REVIEW

**Comment by flipog** [comment]
I forgot a typo in line 194 where 'resistent' should be 'resistant'

**Comment by MarcoWorms** [comment]
Can you correct the typo please? We can't change your branch :/

File: src/game/models/equips.js [hunk 1]
```diff
@@ -310,7 +310,7 @@ export default [
   {
     id: 'ice_ring' ,
     name: 'Ice Ring',
-    description: 'It looks as powerfull as it shines.',
+    description: 'Ice rink?! Imagine how cray, da &^@#ing szie fo dose s#!%!!!1!',
```
Comment (MarcoWorms) [inline src/game/models/equips.js]: didn't get this one hahahaha

Comment (flipog) [inline src/game/models/equips.js]: I fixed the typo, and this joke is about the "narrator" understanding wrong what he was supposed to describe, but given it had to be explained, I imagine it should be changed?

Comment (flipog) [inline src/game/models/equips.js]: The typos left (including this bit) are on purpose. Tried to keep it lightweight

Comment (MarcoWorms) [inline src/game/models/equips.js]: Thanks! tbh i don't think i like the intensive use of slang :/

**Pull request merged**